packages/itwin/reports-config-widget/src/test/ReportMappings.test.tsx

```diff
@@ -419,289 +384,40 @@ describe("Report Mappings View", () => {
     ).toBeInTheDocument();
   });
 
-  it("remove a report mapping", async () => {
```

Comment:
Why did you remove the `remove a report mapping` test?

Reply:
`remove a report mapping` test has been moved to `ReportMappingHorizontalTile` tests, since the only way to trigger deletion of Report Mapping is through `HorizontalTile`.